lib/ui/settings/data/import/analyze_qr_code.dart

```diff
@@ -0,0 +1,260 @@
+                          );
+                          if (result?.action != null &&
+                              result!.action == ButtonAction.first) {
+                            isScannedByImage = false;
+                          }
+                        }
+                        HapticFeedback.vibrate();
+                        List<Code> codes =
+                            parseGoogleAuth(capture.barcodes[0].rawValue!);
+                        scannerController.dispose();
+                        Navigator.of(context).pop(codes);
+                      } else {
+                        showToast(context, "Invalid QR code");
+                        isNavigationPerformed = false;
+                      }
+                    }
+                  },
+                ),
+                // Qr code scanner overlay
+                const QRScannerOverlay(),
+
+                // Torch and gallery buttons
+                Positioned(
+                  top: 150,
+                  left: 0,
+                  right: 0,
+                  child: Row(
+                    mainAxisAlignment: MainAxisAlignment.center,
+                    children: [
+                      // Torch button
+                      IconButton(
+                        icon: ValueListenableBuilder(
+                          valueListenable: scannerController.torchState,
+                          builder: (context, state, child) {
+                            switch (state) {
+                              case TorchState.on:
+                                return SvgPicture.asset(
+                                  'assets/scanner-icons/icons/flash_on.svg',
+                                );
+                              case TorchState.off:
+                                return SvgPicture.asset(
+                                  'assets/scanner-icons/icons/flash_off.svg',
+                                );
+                            }
+                          },
+                        ),
+                        iconSize: 60,
+                        onPressed: () => scannerController.toggleTorch(),
+                      ),
+
+                      // Gallery button
+                      IconButton(
+                        icon: SvgPicture.asset(
+                          'assets/scanner-icons/icons/gallery.svg',
+                        ),
+                        iconSize: 60,
+                        onPressed: () async {
+                          final result = await showDialogWidget(
+                            context: context,
+                            title: l10n.importFromApp(
+                              "Google Authenticator (saved image)",
+                            ),
+                            body:
+                                'Please turn off all photo cloud sync from all apps, including iCloud, Google Photo, OneDrive, etc. \nAlso if you have a second smartphone, it is safer to import by scanning QR code.',
```

Comment:
Can you please extract the newly added strings and put them in the arb file?

Reply:
Is the Strings finalized, and how to extract it, like od I need to add to every arb file ?

Reply:
It'll be easier to review the strings if you extract them to a single file. These need to be extracted to `app_en.arb`, rest of the translations are contributed by the community via [CrowdIn](https://crowdin.com/).